generated/nidcpower/session.py

```diff
@@ -3073,24 +3070,21 @@ def query_in_compliance(self):
         The compliance limit is the current limit when the output method is
         set to OutputFunction.DC_VOLTAGE. If the output is operating at the
         compliance limit, the output reaches the current limit before the
-        desired voltage level. Refer to the ConfigureOutputFunction
-        method and the ConfigureCurrentLimit method for more
+        desired voltage level. Refer to the configure_output_function
```

Comment:
This is an odd side effect of the change... before we'd use the C function name minus the prefix. Now we use the Python name of a non-existent method.

Was another side effect of this making the "funky" warnings from  #489 go away? I don't think that's desirable - the warnings are effectively masked but the problem they originally pointed out is still there. If that's the case I think we need a bit of further discussion.

Reply:
A very long explanation why this changed.

* Metadata on GitHub currently includes functions with `'codegen_method': 'no'`
* When I update the metadata soon (I am currently working on this) then those functions will no longer be added to the metadata on GitHub and will just go away.
* Before this change, we only expanded functions that were in the API (I.e. part of `Session`).
* After the change, we need to expand functions that are not part of the API, but are also _not_ `'codegen_method': 'no'`. I.e. `'library-only'`

I could have modified the code generator even more to avoid these changes, and I would have if the changes would have become permanent, but they aren't.

Once the metadata is updated without the unused functions, the documentation (and the note) will go back to the way they were.

It seemed foolish to torque the code generator around just to avoid temporary changed to the documentation.

Reply:
I don't fully follow why we need `libary-only` now but in the future the metadata on GitHub won't contain those methods at all.

#1103 created to track this temporary problem.